### DISCUSSION TITLE
Is there a way to create an If node like structure?

### DISCUSSION THREAD

**Discussion by talhaamjadx** [bcakmakoglu/vue-flow#1544] [Q&A]
![image](https://github.com/user-attachments/assets/ba19b6de-585f-43d3-b395-4fed1b20f68b)

I want to create something like a flow diagram. Right now I have a separate svg inside a custom if node. Is there a way to add something similar using the vue flow utilities? The if can be nested as well, like having if as a child inside a parent if node.

**Comment by 0x7A7A6572** [comment]
I've encountered a similar issue. Have you managed to resolve it?
